CHANGELOG.md

```diff
@@ -269,15 +271,11 @@ try (final @NotNull ISentryLifecycleToken ignored = Sentry.pushIsolationScope())
   // this block has its separate isolation scope
 }
 ```
+- Classes used by our previous OpenTelemetry integration have been deprecated (`SentrySpanProcessor`, `SentryPropagator`, `OpenTelemetryLinkErrorEventProcessor`). Please take a look at [docs](https://docs.sentry.io/platforms/java/tracing/instrumentation/opentelemetry/) on how to setup OpenTelemetry in v8.
 
 You may also use `LifecycleHelper.close(token)`, e.g. in case you need to pass the token around for closing later.
 
 
-### Migration Guide
-
-Please take a look at [our migration guide in docs](https://docs.sentry.io/platforms/java/migration/7.x-to-8.0).
-
-
 ### Changes from `rc.4`
```

Comment:
Do we need the "Changes from `rc.4`" part - as we don't have other pre-release sections?